src/upgrades.d/00001.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace upgrade1;
+
+$current_version = getDefaultsVersion();
```

Comment:
We could move it in the general script: it looks to be useful in the future...

Reply:
can I use it if it is in a different namespace?

Reply:
It's required a namespace prefix in that case, or to import it with "use"